src/github/dto/update.practices.dto.ts

```diff
@@ -0,0 +1,11 @@
+import { IsNotEmpty } from 'class-validator';
+import { ApiProperty } from '@nestjs/swagger';
+
+export class UpdatePracticesDto {
+  @ApiProperty({
+    description: 'Practices can be strings or a single paragraph.',
+    example: ['practice1', 'practice2'],
+  })
+  @IsNotEmpty()
+  practices: string[];
+}
```

Comment:
PullProofAI

The `UpdatePracticesDto` class looks well-defined with proper usage of `class-validator` for validation and `@nestjs/swagger` for API documentation. It's good to see clear descriptions and examples provided for the `practices` property.

Overall, this new hunk is well-structured and adheres to best practices for defining DTO classes in a NestJS application.



<!-- This is an auto-generated comment by PullProofAI -->